scipy/interpolate/_bsplines.py

```diff
@@ -795,8 +795,10 @@ def make_interp_spline(x, y, k=3, t=None, bc_type=None, axis=0,
 
     y = np.rollaxis(y, axis)    # now internally interp axis is zero
 
-    if x.ndim != 1 or np.any(x[1:] <= x[:-1]):
+    if x.ndim != 1 or np.any(x[1:] < x[:-1]):
```

Comment:
Well I guess another option is to change the original message to: `Expect x to be 1-dimensional and strictly increasing`.

Reply:
That would be another simpler option. The only downside is that less experienced developers might be less familiar with that term.

Reply:
You can chose either option, but as far as I remember in interpolator classes there are messages similar to what I mentioned.